openedx_proversity_reports/edxapp_wrapper/get_courseware_library.py

```diff
@@ -0,0 +1,12 @@
+""" Backend abstraction. """
+from importlib import import_module
+from django.conf import settings
+
+
+def get_course_by_id(*args, **kwargs):
+    """ Get the course for the given course id. """
```

Comment:
.

Reply:
Let's use the first pearson convention in this file.